synthtool/gcp/common.py

```diff
@@ -353,3 +359,9 @@ def _load_repo_metadata(metadata_file: str = "./.repo-metadata.json") -> Dict:
         with open(metadata_file) as f:
             return json.load(f)
     return {}
+
+def _get_default_branch_name(package_name: str) -> str:
```

Comment:
It looks like this might not be used yet?

Reply:
fixed